web/sites/default/config/field.field.group.base_group.field_group_path.yml

```diff
@@ -15,7 +15,7 @@ field_name: field_group_path
 entity_type: group
 bundle: base_group
 label: 'Group path'
-description: "<ul>\r\n<li>Use group path to set the URL for your group and its content.</li>\r\n<li>Paths should follow the pattern of [parent-office]/[parent-program]/[advisory-group]. For example, bps/cleanenergy/bylaws-subcommittee.</li>\r\n<li>Including a parent in the group path will automatically add that group into the breadcrumbs for this group.</li>\r\n<li><strong>Update with caution.</strong> Updating a group path will update the paths for all content within the group as a bulk operation.</li>\r\n</ul>"
+description: "<ul>\r\n<li>Use group path to set the URL for your group and its content.</li>\r\n<li>Paths should follow the pattern of [parent-office]/[parent-program]/[advisory-group]. For example, bps/cleanenergy/bylaws-subcommittee.</li>\r\n<li>Including a parent in the group path will automatically add that group into the breadcrumbs for this group.</li>\r\n<li><strong>Update with caution.</strong> Updating a group path will update the paths for all content within the group as a bulk operation.</li>\r\n<li>The group path is only accessible by members with the Group Creator, Sitewide Editor or Sitewide Administrator role.</li>\r\n</ul>"
```

Comment:
Could we add another bullet point to this field on both group types: "Do not add a leading or trailing slash."